pq-crypto/sike_r2/sike_r2_code_identifier.h

```diff
@@ -20,6 +20,9 @@
 // Allow the forced usage of the generic implementation if desired.
 #if defined(S2N_SIKE_R2_FORCE_GENERIC)
 #define GENERIC_IMPLEMENTATION
+// The ASM code isn't written for OSX; must use generic
+#elif defined(__APPLE__)
+#define GENERIC_IMPLEMENTATION
```

Comment:
Should we give this a prefixed name, to avoid future name conflicts?

Reply:
Done